src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -120,12 +120,12 @@
      * URL of JIRA for normal access, like <tt>http://jira.codehaus.org/</tt>.
      * Mandatory. Normalized to end with '/'
      */
-    public final URL alternativeUrl;
+    public URL alternativeUrl;
```

Comment:
this still should be final, right?

Reply:
In Jenkins terms, final is used for mandatory values.
if it were Mandatory it should be final and be in the constructor.

see:
https://jenkins.io/doc/developer/plugin-development/pipeline-integration/#constructor-vs-setters